source/refined-github.ts

```diff
@@ -20,6 +20,7 @@ import './features/sticky-file-header.css';
 import './features/readable-title-change-events.css';
 import './features/clean-checks-list.css';
 import './features/sticky-csv-header.css';
+import './features/mark-private-repos.css';
```

Comment:
While most sections of features are sorted, I noticed that the list of CSS-only features was not sorted. So I just put mine at the end.

I'm happy to push up a commit sorting the list of CSS-only features if desired.

Reply:
They're not sorted, we just append features to the list, so this was the right choice.